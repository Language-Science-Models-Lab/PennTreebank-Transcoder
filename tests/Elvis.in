
(S (NP @(NNP @Elvis)) @(VP @(VBZ @is) (ADJP-PRD~ @(JJ @dead))) (. @.))

# tests phrase rules 1 and 13
# tests clause rule 1

# 'dead' should be parsed by 
# phrase : '(' nonterminal word ')' { $$ = $3 ; } ;

# 'is dead' should be parsed by
# phrase : '(' nonterminal word phrase ')' { $$ = output( "[%s %s]", 2, $3, $4 ) ; } ;

# 'Elvis' should be parsed by
# phrase : '(' nonterminal word ')' { $$ = $3 ; } ;

# 'Elvis is dead' should be parsed by
# clause : '(' s phrase headphrase ')' { $$ = output( "[%s %s]", 2, $3, $4 ) ; } ;

