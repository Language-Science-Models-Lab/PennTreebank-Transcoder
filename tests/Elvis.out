[Elvis, [is, dead]]:type(type(type(is, _)@type(dead, _), _)@type(Elvis, _), s)
